transcoders.go

```diff
@@ -391,3 +391,33 @@ func certHashStB(s string) ([]byte, error) {
 func certHashBtS(b []byte) (string, error) {
 	return multibase.Encode(multibase.Base64url, b)
 }
+
+var TranscoderHostHeader = NewTranscoderFromFunctions(hostHeaderStB, hostHeaderBtS, hostHeaderVal)
+
+func hostHeaderVal(b []byte) error {
+	split := bytes.Split(b, []byte{':'})
```

Comment:
It’s probably easier to convert to string once in the beginning. I think there’s also a standard library function to split a hostname and a port. I don’t remember how it’s called though.

Reply:
the way you phrased it pointed me in the right direction. There is [net.SplitHostPort](https://pkg.go.dev/net#SplitHostPort). I have updated the function accordingly.